app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class LikesController < ApplicationController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable '@likable' [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->